docs/discovery/webfinger.md

```diff
@@ -6,7 +6,7 @@ diaspora\* uses an old draft of [WebFinger][webfinger-draft] to discover users f
 
 {% include warning_box.html
    title="Old WebFinger"
-   content="<p>diaspora* doesn't yet support the RFC 7033 WebFinger!</p>"
+   content="<p>diaspora* doesn't yet fully support the RFC 7033 WebFinger!</p>"
```

Comment:
> `diaspora* uses an old draft of WebFinger to discover users from other pods.`

This is still true, because diaspora still uses the old WebFinger for discovery, this PR adds support that we provide the new RFC 7033, so we can use it later (and other software can use it).

> Also we have "Additional information and specifications" box at the page footer. We could actually add RFC 7033 WebFinger specification link there.

👍